crates/control/src/behavior/defend.rs

```diff
@@ -107,6 +137,38 @@ fn defend_left_pose(
     Some(robot_to_field.inverse() * defend_pose)
 }
 
+fn defend_penalty_left_pose(
+    world_state: &WorldState,
+    field_dimensions: &FieldDimensions,
+    role_positions: &RolePositions,
+) -> Option<Isometry2<f32>> {
+    let robot_to_field = world_state.robot.robot_to_field?;
+    let ball = world_state
+        .ball
+        .map(|ball| BallState {
+            position: point![
+                -field_dimensions.length / 2.0 + field_dimensions.penalty_marker_distance,
+                0.0
+            ],
+            field_side: ball.field_side,
+            penalty_shot_direction: Default::default(),
+        })
+        .unwrap_or_default();
+
+    let position_to_defend = point![
+        (-field_dimensions.length + field_dimensions.penalty_area_length) / 2.0,
+        role_positions.defender_y_offset
+    ];
```

Comment:
Compared to the regular defending position which is just -field_dimensions.length / 2.0 this is offset by half the penalty area length and essentially works as an offset for the defenders so that they remain outside the penalty box

Reply:
changed to increased defensive radius as discussed